card/src/main/java/com/adyen/checkout/card/StoredCardDelegate.kt

```diff
@@ -171,6 +178,46 @@ internal class StoredCardDelegate(
         }
     }
 
+    @Suppress("TooGenericExceptionCaught")
```

Comment:
Why do we need to suppress this?

Reply:
Was related to NullPointerException, removed it as it's not thrown anymore